benchmark/benchmark.py

```diff
@@ -274,19 +293,26 @@ def torchvision_transform(self, img):
     def keras(self, img):
         return keras.apply_brightness_shift(img, brightness=1.5).astype(np.uint8)
 
+    def kornia_op(self, img):
+        return K.functional.apply_adjust_brightness(img, self.kornia_brightness_params)
+
 
 class Contrast(BenchmarkTest):
     def __init__(self):
         self.imgaug_transform = iaa.Multiply((1.5, 1.5), per_channel=False)
         self.augmentor_op = Operations.RandomContrast(probability=1, min_factor=1.5, max_factor=1.5)
         self.solt_stream = slc.Stream([slt.ImageRandomContrast(p=1, contrast_range=(1.5, 1.5))])
+        self.kornia_contrast_params = {"contrast_factor": torch.Tensor([1.5])}
```

Comment:
better to use `torch.tensor`. `torch.Tensor` is not recommended to create tensors